auth4genai/snippets/mcp/get-started/server-authorization.mdx

```diff
@@ -23,60 +23,123 @@ By the end of this quickstart, you will have an MCP server that:
 
 <AssignPermissionsToRoles />
 
+## Sample app
 <Tabs>
-  <Tab title="Use sample app (recommended)">
-    Start by downloading the sample app for this quickstart. The sample includes a FastMCP MCP server with an Auth0 integration in JavaScript.
-
-    <DownloadQuickstartButton
-      category="auth-for-mcp"
-      framework="fastmcp-mcp-js"
-    />
-
-    Once downloaded, extract the files and open the project in your preferred IDE.
-</Tab>
-  <Tab title="Clone GitHub repository">
-    Clone the repository and navigate to the sample app folder which includes a FastMCP MCP server with an Auth0 integration in JavaScript.
-
-    ```shell wrap lines
-    git clone https://github.com/auth0-samples/auth0-ai-samples.git
-    cd auth0-ai-samples/auth-for-mcp/fastmcp-mcp-js
+  <Tab title="Javascript" icon="js">
+    <Tabs>
+      <Tab title="Use sample app (recommended)">
+        Start by downloading the sample app for this quickstart. The sample includes a FastMCP MCP server with an Auth0 integration in JavaScript.
+
+        <DownloadQuickstartButton
+          category="auth-for-mcp"
+          framework="fastmcp-mcp-js"
+        />
+
+        Once downloaded, extract the files and open the project in your preferred IDE.
+      </Tab>
+      <Tab title="Clone GitHub repository">
+        Clone the repository and navigate to the sample app folder which includes a FastMCP MCP server with an Auth0 integration in JavaScript.
+
+        ```shell wrap lines
+        git clone https://github.com/auth0-samples/auth0-ai-samples.git
+        cd auth0-ai-samples/auth-for-mcp/fastmcp-mcp-js
+        ```
+
+        Once cloned, open the project in your preferred IDE.
+      </Tab>
+    </Tabs>
+    ## Install packages
+
+    Ensure you have npm installed or follow the instructions to [install npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) in its documentation. In the `fastmcp-mcp-js` directory, install the required packages:
+
+    ```shell
+    npm install
     ```
 
-    Once cloned, open the project in your preferred IDE.
+    ## Create your environment file
+
+    In the `fastmcp-mcp-js` directory, create a new `.env` file and add the following content:
+
+    ```shell
+    AUTH0_DOMAIN=YOUR_AUTH0_DOMAIN
+    AUTH0_AUDIENCE=http://localhost:3001
```

Comment:
Should we fix the `.env.example` file in the sample [here](https://github.com/auth0-samples/auth0-ai-samples/blob/0d13b41203b4e67c3f56bf157d1a07aaa80fe0cb/auth-for-mcp/fastmcp-mcp-python/.env.example#L7)? (set the default value to `http://localhost:3001`)